Src/ResistanceOperationOverhaul/Classes/UITLE_LadderModeMenu_Override.uc

```diff
@@ -134,7 +134,7 @@ simulated function InitScreen(XComPlayerController InitController, UIMovie InitM
 	}
 
 	// Start with the default squad - 4 random soldiers at the start, with others joining at missions 2 and 4
-	Soldier.bRandomlyGeneratedCharacter = true;
+	Soldier.bRandomCharacter = true;
```

Comment:
Please revert the changes to this file - I would prefer that randomly generated characters be the default.